.github/workflows/CI.yml

```diff
@@ -94,7 +93,32 @@ jobs:
       with:
         name: binary_artifacts
         path: build
+        
+  analyze:
+    name: CodeQL Analyze
+    needs: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
 
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v1
+        with:
+          languages: "go"
+        
+      - name: Download built artifacts
```

Comment:
Why do we need to download the artifacts?

Reply:
Hmm, I think by default, some languages must be built and compiled for the code scanning (more specifically for languages like Java and C++). I wasn't sure if we needed it for Golang (but I haven't found any documentation on it). I'll try testing without it and seeing if the code scanning still runs

Reply:
Seems like we actually don't need it. I'll remove this and push again.